src/ape_accounts/accounts.py

```diff
@@ -257,3 +259,39 @@ def __decrypt_keyfile(self, passphrase: str) -> HexBytes:
             return EthAccount.decrypt(self.keyfile, passphrase)
         except ValueError as err:
             raise InvalidPasswordError() from err
+
+
+def generate_account(
+    alias: str, passphrase: str, hd_path: str = ETHEREUM_DEFAULT_PATH, word_count: int = 12
```

Comment:
Maybe `passphrase` can default to `None` to expose a way to take interactive input (using private password input from click) if it's not specified. It'll also look nicer because the kwarg `passphrase="something"` is a little more explicit for scripting use

Reply:
Using click in library functions seems very unexpected.  The CLI was refactored to ask for passphrase and feed it to this function.

Reply:
Sure, not a showstopper either way